### DISCUSSION TITLE
Feature request: Add the ability to customize the existing fields in user forms/tables

### DISCUSSION THREAD

**Discussion by evdmeer** [tomatophp/filament-users#22] [Ideas]
I would love to have the ability to change/customize existing fields in the user tables and forms.

Example 1: I would like the change the password field to include a generate password action that generates a random password. 
Example 2: I would like to customize how the 'roles' column is displayed in the users table. I want to add colored icons per role instead of the standard badges.

I would be nice if there would be a way that allows me to get the current schema of the form/table, adjust any parameters on the components and set that new schema as the current one.

Either returning the component array, or getting and setting by column/field name would be nice.
This would also allow for easy mxing/reordering of custom and default fields; it would allow custom fields to be displayed before or in between the default ones.

**Comment by fadymondy** [comment]
your feature is ready on the new release


https://github.com/tomatophp/filament-users/releases/tag/v2.0.8
